test/e2e/run_e2e.sh

```diff
@@ -45,8 +46,15 @@ CDP_TARGET_COMMIT_ID="${CDP_TARGET_COMMIT_ID:-"dev"}"
 CDP_HEAD_COMMIT_ID="${CDP_HEAD_COMMIT_ID:-"$(git describe --tags --always)"}"
 RESULT_BUCKET="${RESULT_BUCKET:-""}"
 
-export CLUSTER_ALIAS="${CLUSTER_ALIAS:-"e2e-${CDP_BUILD_VERSION}"}"
-export LOCAL_ID="${LOCAL_ID:-"e2e-${CDP_BUILD_VERSION}"}"
+if [ "$OPA_ENABLED" == "true" ]; then
```

Comment:
this is a matter of taste, so you can ignore

instead of using `OPA_ENABLED`

add 2 environment variables one for FOCUS which defaults to the current value and another for cluster suffix, then you can 

```sh
FOCUS="${FOCUS:-(\[Conformance\]|\[StatefulSetBasic\]|\[Feature:StatefulSet\]\s\[Slow\].*mysql|\[Zalando\])}"
export CLUSTER_ALIAS="${CLUSTER_ALIAS:-"e2e-${CDP_BUILD_VERSION}${CLUSTER_SUFFIX}"}"
export LOCAL_ID="${LOCAL_ID:-"e2e-${CDP_BUILD_VERSION}${CLUSTER_SUFFIX}"}"
```